src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+
+import duke.tasks.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Container for a list of task items.
+ */
+public class TaskList {
+    private List<Task> items;
+
+    /**
+     * Instantiates a task list object.
+     * @param items list of items to be stored in the task list.
+     */
+    public TaskList(List<Task> items) {
+        this.items = items;
+    }
+
+    /**
+     * Returns the task at the specified index.
+     *
+     * @param index index of the task specified.
+     * @return task at the specified index.
+     */
+    public Task getTask(int index) {
+        return items.get(index);
+    }
+
+    /**
+     * Returns the list of task items.
+     *
+     * @return list of task items.
+     */
+    public List<Task> getItems() {
+        return items;
+    }
+
+    /**
+     * Return the number of task items in the list.
```

Comment:
Similarly to your other javadocs headers (which are well done! 😺 ), headers should start with words in the form `Returns ...`, `Sends ...`, `Adds ...` (not `Return` or `Returnning` etc.)